### ISSUE TITLE
investigate GNN metacontroller crash

### ISSUE THREAD

**Issue by tomsilver** [Learning-and-Intelligent-Systems/predicators#809]
example:
```python src/main.py --env stick_point --min_perc_data_for_nsrt 1 --segmenter oracle --num_train_tasks 50 --timeout 300 --seed 458 --gnn_num_epochs 5000 --experiment_id gnn_metacontroller_50 --approach gnn_metacontroller --option_learner direct_bc_nonparameterized```

on commit 385de97cf44cdbbc13667a0847fd7396a36b2fbc

leads to
```
Traceback (most recent call last):
  File "/home/gridsan/tslvr/predicators/src/main.py", line 361, in <module>
    main()
  File "/home/gridsan/tslvr/predicators/src/main.py", line 109, in main
    _run_pipeline(env, approach, stripped_train_tasks, offline_dataset)
  File "/home/gridsan/tslvr/predicators/src/main.py", line 131, in _run_pipeline
    approach.learn_from_offline_dataset(offline_dataset)
  File "/home/gridsan/tslvr/predicators/src/approaches/gnn_metacontroller_approach.py", line 227, in learn_from_offline_dataset
    GNNApproach.learn_from_offline_dataset(self, dataset)
  File "/home/gridsan/tslvr/predicators/src/approaches/gnn_approach.py", line 121, in learn_from_offline_dataset
    data = self._generate_data_from_dataset(dataset)
  File "/home/gridsan/tslvr/predicators/src/approaches/gnn_metacontroller_approach.py", line 61, in _generate_data_from_dataset
    target = self._extract_target_from_segment(segment)
  File "/home/gridsan/tslvr/predicators/src/approaches/gnn_metacontroller_approach.py", line 74, in _extract_target_from_segment
    seg_option = segment.get_option()
  File "/home/gridsan/tslvr/predicators/src/structs.py", line 1107, in get_option
    assert self.has_option()
AssertionError
```

**Comment by ronuchit** [comment]
this is closed by #810 right @tomsilver

**Comment by tomsilver** [comment]
yessir
